index.tt

```diff
@@ -6,152 +6,349 @@
+            to build docker images</a>.
+        </p>
+
+      </div>
+      <div class="span6">
+
+        <h3>Declarative cloud images</h3>
+        <p>How hard would it be to build and configure <strong>Amazon AMI?</strong></p>
+        <p>
+          With the following <code>amazon.nix</code> we defined nginx which is
+          serving example <code>/var/www</code> folder, having a valid ssl
+          certificate (via LetsEncrypt) and enabled recommended security
+          settings.
+        </p>
+        <pre class="well" style="margin-top:1em">
+{ pkgs, ...}:
+{
+  security.acme.acceptTerms = true;
+  security.acme.email = "nix@example.com";
+  services.nginx = {
+    enable = true;
+    recommendedGzipSettings = true;
+    recommendedOptimisation = true;
+    recommendedProxySettings = true;
+    recommendedTlsSettings = true;
+    virtualHosts."example.com" = {
+      enableACME = true;
+      forceSSL = true;
+      locations."/".root = "/var/www";
+    };
+  };
+}</pre>
+        <p>Now we just need to build it and upload it.</p>
+        <pre class="well" style="margin-top:1em">
+<span class="nix-command"><strong>nix-build '&lt;nixpkgs/nixos/release.nix&gt;' -A amazonImage.x86_64-linux --arg configuration ./amazon.nix</strong></span>
+...
+/nix/store/wmg6gcgnh6bk8d98syydfq73q6hjv682-nixos-amazon-image-20.09pre130979.gfedcba-x86_64-linux
+<span class="nix-command">ls -lh ./result/</span>
+-r--r--r--     1 root root   1.4G Jan  1  1970 nixos-amazon-image-20.09pre130979.gfedcba-x86_64-linux.vhd
+dr-xr-xr-x     2 root root      4 Jan  1  1970 nix-support
+<span class="nix-command"><strong>nix-shell -p awscli</strong></span>
+(nix-shell)<span class="nix-command">export AWS_IMAGE="./result/nixos-amazon-image-20.09pre130979.gfedcba-x86_64-linux.vhd"</span>
+(nix-shell)<span class="nix-command">aws s3 cp --region $REGION $AWS_IMAGE "S3://${BUCKET}/${AWS_IMAGE#/}"</span>
+...</pre>
+
+      </div>
     </div>
-    <div class="span4 and-more">
-      <h3>About the community</h3>
-      <p>
-        Sometimes, it helps to talk with someone.  Our community
-        is there whether you want to help or get support.
-      </p>
-      <ul>
-        <li>
-          <a href="[%root%]nixos/community.html">
-            Get in touch
-            <i class="fa fa-angle-right"></i>
-          </a>
-        </li>
-        <li>
-          <a href="[%root%]learn.html">
-            Learning resources
-            <i class="fa fa-angle-right"></i>
-          </a>
-        </li>
-      </ul>
+  </section>
+  <section class="tab-pane" id="deploy">
+    <div class="row-fluid">
+      <div class="span6">
+        <h3>TODO: Ideas</h3>
+        <ul>
+          <li>How to integrate with CI (travis, github actions, buildkite, ...) but that is notreally a deployment</li>
+        </ul>
+      </div>
+      <div class="span6">
+        <h3>NixOps</h3>
+        <p>TODO: can not come with a good example that is simple enough</p>
```

Comment:
you could do the exact same as with AMI example, but that might be lame

check their manual. some example which setups multiple servers, like 2 or 3 with virtualbox. or better nixos containers as virtualbox might not be installed. explain you could deploy the same servers to X, Y, Z...

Reply:
for now i'm skipping this examples. we can add them later.